LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentAuthResultInfo.java

```diff
@@ -0,0 +1,18 @@
+package com.braintreepayments.api;
+
+
+/**
+ * Details of a {@link LocalPaymentAuthResult.Success}
+ */
+public class LocalPaymentAuthResultInfo {
```

Comment:
This class can be removed and `LocalPaymentAuthResult` can be updated to have an `internal val browserSwitchResultInfo` as a param, but first we need to convert `LocalPaymentClient` to Kotlin because the java/kotlin interop with internal vals doesn't work well. Same applies for PayPal and SEPA. Created a separate ticket for that.